src/lib/workspace/AnalysisList.tsx

```diff
@@ -14,16 +20,11 @@ export function AnalysisList(props: Props) {
   const { analysisStore } = props;
   const studyRecord = useStudyRecord();
   const studyId = studyRecord.id.map((part) => part.value).join('/');
-  const [analysisList, setAnalysisList] = React.useState<Analysis[]>();
   const history = useHistory();
   const [selected, setSelected] = React.useState<Set<string>>(new Set());
-  const updateAnalysisList = React.useCallback(async () => {
-    const list = await analysisStore.getAnalyses();
-    setAnalysisList(list.filter((a) => a.studyId === studyId));
-  }, [analysisStore, studyId]);
-  React.useEffect(() => {
-    updateAnalysisList();
-  }, [updateAnalysisList]);
+  const { analyses: analysisList, deleteAnalyses } = useAnalysisList(
+    analysisStore
+  );
```

Comment:
Need to reinstate the filter on studyId 
```suggestion
  const { analyses, deleteAnalyses } = useAnalysisList(
    analysisStore
  );
  const analysisList = analyses ? analyses.filter((analysis) => analysis.studyId === studyId) : undefined;
```